### ISSUE TITLE
Check the calculator integration with mkdocs site

### ISSUE THREAD

**Issue by sei-vsarvepalli** [CERTCC/SSVC#412]

**Comment by sei-vsarvepalli** [comment]
Temporary development site is in private fork with a public Pages export. Have a look at 
https://sei-vsarvepalli.github.io/SSVC/ssvc-calc/

**Comment by ehatleback** [comment]
In the first decision point, scrolling over the options highlights the top and bottom options (None and Active) in red, but the middle option (PoC) does not include the red highlighting.

Additionally, I'd suggest that the appearance of the options when I hover is somehow a bit jarring or disorienting.  A better option might be for the available selections to either be available all the time on the screen (rather than appearing when you hover), or to appear when you hover, but always to appear in the same location on the screen, like at the bottom.  

Finally, the options that appear when you hover frequently look like they're a little bit too big for the window they appear in; the text is right at the very edge of, and sometimes even cut off a bit from, the box that contains the appearing text.

I'm on Chrome on a Macbook, if that helps contextualize what I'm seeing.

**Comment by ahouseholder** [comment]
In Chrome Version 120.0.6099.234 (Official Build) (x86_64), I currently see this:
<img width="1282" alt="Screenshot 2024-01-31 at 3 36 56 PM" src="https://github.com/CERTCC/SSVC/assets/2594236/54848c9f-8dab-4108-97e4-1d2a68cd5133">

**Comment by ahouseholder** [comment]
That might have been because I had the JS cached though. On a page reload it looks fine.

**Comment by ehatleback** [comment]
> That might have been because I had the JS cached though. On a page reload it looks fine.

Yes, the very same thing happened to me.

**Comment by ahouseholder** [comment]
I copied @ehatleback's [comment](https://github.com/CERTCC/SSVC/issues/412#issuecomment-1919764611) and @sei-bkoo's [comment](https://github.com/CERTCC/SSVC/issues/412#issuecomment-1920124439) to a new issue:

- #438 

...in case these turn out to be too far out of scope for the current issue. If we happen to do a PR that resolves both #412 and #438 at the same time, great. But I wanted to make sure that #412 is resolvable independently as long as the existing SSVC-calc works in the context of the new site design. The UI enhancements in the two comments above might turn out to be out of scope for that, so I figured two issues gives us the option of resolving one now and the other later if we so choose.
